.travis.yml

```diff
@@ -0,0 +1,93 @@
+language: php
+
+sudo: false
+
+env:
+  global:
+    - WP_PHPUNIT=phpunit
+
```

Comment:
You are right, I've updated the `.travis.yml` file to disable XDebug when necessary. I also disabled the message (TravisCI trigger itself a composer update before the CI process starts...).

Thank you for the feedback 😄 !